setup.py

```diff
@@ -216,7 +216,8 @@
             'view_restricted_collection'
             ' = inspirehep.modules.records.permissions:'
             'action_view_restricted_collection',
-            'admin_holdingpen_authors = inspirehep.modules.authors.permissions:action_admin_holdingpen_authors'
+            'admin_holdingpen_authors = inspirehep.modules.authors.permissions:action_admin_holdingpen_authors',
+            'editor_manage_tickets = inspirehep.modules.editor.permissions:action_editor_manage_tickets'
```

Comment:
Nit: this example shows why putting trailing commas everywhere is better, because it leads to simpler diffs (unlike what happened here).